CHANGELOG.md

```diff
@@ -23,6 +23,11 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.21.0...master
 
+### Fixed
+- Do not explicitly specify Availability Zone when requesting resources to reduce insufficient capacity errors. Resolve [JPERF-637].
```

Comment:
Nit, a purely stylistic remark:
I try to specify what the code *does* rather than what it *doesn't do*.
E.g.:
```suggestion
- Let AWS pick the Availability Zone to balance resource capacity. Resolve [JPERF-637].
```